src/test/ui/feature-gate-non_ascii_idents.stderr

```diff
@@ -83,22 +79,16 @@ error: non-ascii idents are not fully supported. (see issue #28979)
 error: non-ascii idents are not fully supported. (see issue #28979)
   --> $DIR/feature-gate-non_ascii_idents.rs:33:1
    |
-33 | / enum Bär { //~ ERROR non-ascii idents
-34 | |     Bäz { //~ ERROR non-ascii idents
-35 | |         qüx: isize //~ ERROR non-ascii idents
-36 | |     }
-37 | | }
-   | |_^
+33 | enum Bär { //~ ERROR non-ascii idents
+   | ^^^^^^^^
    |
    = help: add #![feature(non_ascii_idents)] to the crate attributes to enable
 
 error: non-ascii idents are not fully supported. (see issue #28979)
   --> $DIR/feature-gate-non_ascii_idents.rs:34:5
    |
-34 | /     Bäz { //~ ERROR non-ascii idents
-35 | |         qüx: isize //~ ERROR non-ascii idents
```

Comment:
Why doesn't this line give an error?

Reply:
Good question. It wasn't already, would have to check.